packages/core/src/router.tsx

```diff
@@ -529,172 +666,328 @@ export async function PageInit<T>({
    *
    **/
 
+  router.addRoute(
+    "/admin",
+    async () => {
+      const headersList = headers()
+      const userAgent = headersList.get("user-agent")
+      const isMobile = /mobile/i.test(userAgent ?? "")
 
+      if (!isMobile) {
+        redirect("/admin/site")
+      }
```

Comment:
Problem is this only works on mobile devices, and not on desktop and responsive

Reply:
Fixed